### PR TITLE
4182 On Explorer, disable ACS source for community districts and blocks

### PR DESCRIPTION
**Pull request by dhochbaum-dcp**
### Summary
Updated sources drop-down to hide ACS options if the geotype is districts or blocks.

#### Tasks/Bug Numbers
 - Completes [AB#4182](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/4182)

### Technical Explanation
This pulls the geoType from this.model.selection.type (which always shows the geoType) rather than this.model.geotype (which returns 'selection' if more than one geo is selected).

### CODE REVIEW

File: app/templates/components/orange-bar-data-explorer.hbs [hunk 1]
```diff
@@ -8,6 +8,7 @@
         @sources={{@sources}}
         @setSource={{@setSource}}
         @isModelLoading={{@isModelLoading}}
+        @geoType={{@geoType}}
```
Comment (godfreyyeung) [inline app/templates/components/orange-bar-data-explorer.hbs:11]: nit: prefer all lowercase `geotype` for consistency with rest of codebase

File: app/components/explorer/source-select-dropdown.js [hunk 2]
```diff
@@ -29,4 +29,11 @@ export default class SourceSelectDropdownComponent extends Component {
   get acsSources() {
     return this.args.sources.filter(source => source.type === 'acs');
   }
+
+  get showACS() {
```
Comment (godfreyyeung) [inline app/components/explorer/source-select-dropdown.js:33]: Nice

**Review by godfreyyeung — APPROVED** [review]
Great catch on using `this.model.selection.type` vs `this.model.geotype`. Thanks for the great Technical explanation writeup.

File: app/templates/explorer.hbs [hunk 3]
```diff
@@ -13,6 +13,7 @@
     @toggleBooleanControl={{this.toggleBooleanControl}}
     @updateCompareTo={{this.updateCompareTo}}
     @isModelLoading={{this.reloadExplorerModel.isRunning}}
+    @geoType={{this.model.selection.type}}
```
Comment (godfreyyeung) [inline app/templates/explorer.hbs:16]: Same here, prefer argument to be `@geotype`.

Comment (dhochbaum-dcp) [reply app/templates/explorer.hbs]: Ok, this has been updated.

**Pull request merged**